firmware/Core/Inc/config/static_config.h

```diff
@@ -0,0 +1,20 @@
+
+#ifndef __INCLUDE_GUARD__STATIC_CONFIG_H__
```

Comment:
Why do we use leading underscores in the include guard? Aren't they reserved for the compiler implementation?

Reply:
You're right, totally forgot about that!

The intention was to put them at the "top" of a sorted list so they're not in the way in Intellisense suggestions. This indeed breaks the rule though.

Whether it's worth it to refactor and re-communicate that change may be open for debate though; the chance of the compiler creating this exact format of macro is probably pretty low (albeit non-zero).